command/daemon/command.go

```diff
@@ -93,6 +93,7 @@ func (c *command) Execute(cmd *cobra.Command, args []string) {
 		serverConfig.ListenAddress = Flags.Server.Listen.Address
 		serverConfig.Logger = c.logger
 		serverConfig.RequestFuncs = customServer.RequestFuncs()
+		serverConfig.Router = customServer.Router()
```

Comment:
The router is now configurable which makes it easy to provide seamless migration paths towards microkit. This was rather easy to accomplish at the end. Back then I struggled conceptually.